libevmasm/Assembly.cpp

```diff
@@ -271,8 +271,8 @@ Json::Value Assembly::assemblyJSON(std::map<std::string, unsigned> const& _sourc
 	{
 		root["sourceList"] = Json::arrayValue;
 		Json::Value& jsonSourceList = root["sourceList"];
-		for (auto const& [name, index]: _sourceIndices)
-			jsonSourceList[index] = name;
+		for (auto const& s: _sourceIndices)
+			jsonSourceList.append(s.first);
```

Comment:
This change was required to prevent `null` entries from being added in the JSON `sourceList` and because of that some tests (for example: `test/cmdlineTests/asm_json`) had their output changed. I'm not sure if the order of the entries in the `sourceList` array matters, but I would guess it doesn't, right?